Improve Concept Exercise: Freelancer Rates (#1472)
exercism/javascript: exercises/concept/freelancer-rates/.docs/instructions.md

# Instructions

> In this exercise you will be writing code to help a freelancer communicate with his clients by providing a few utility functions to quickly calculate his hiring costs for projects.

Reviewer: "hiring costs" sounds like the freelancer hires someone else which is not meant here I think, maybe something like this could be used instead

```suggestion
In this exercise you will be writing code to help a freelancer communicate with his clients about the prices of certain projects. You will write a few utility functions to quickly calculate the costs for the clients.
```

The day rate does not need to be rounded or changed to a "fixed" precision.

> ## 2. Calculate the number of workdays given a fixed budget and hourly rate

Reviewer: The task title does not need to contain all the parameters of the function. Those are described below. The same applies to task 3.
```suggestion
## 2. Calculate the number of workdays given a fixed budget
```

Another day, a project manager offers the freelancer to work on a project with a fixed budget.
Given the fixed budget and the freelancer's hourly rate, help him calculate the number of days the project would take to complete

```javascript
CHANGE THIS EXAMPLE>>>>
daysInBudget(20000, 89);    
// => 28
```

> The result _must_ be rounded down to the nearest whole number.

Reviewer: Sidenote: To address the issue that a lot of people missed the rounding part in this exercise, I would recommend changing the emphasis here:
```suggestion
The result must be **rounded down** to the nearest whole number.
```
Same applies to task 3.

## 3. Calculate the discounted rate, given a day rate, the number of workdays, and a monthly discount rate

> Many a time, the freelancer's clients hire him for projects spanning over months.

Reviewer: ```suggestion
Often, the freelancers' clients hire him for projects spanning multiple months.
```

The day rate does not need to be rounded or changed to a "fixed" precision.

## 2. Calculate the number of workdays given a fixed budget

Another day, a project manager offers the freelancer to work on a project with a fixed budget.

> Given the fixed budget and the freelancer's hourly rate, help him calculate the number of days the project would take to complete

Author: Your suggestion does make sense. I'll commit it :)

```javascript
daysInBudget(20000, 89);
// => 28
```

The result _must_ be **rounded down** to the nearest whole number.

## 3. Calculate the discounted rate for large projects

Often, the freelancer's clients hire him for projects spanning over multiple months.
In these cases, the freelancer decides to offer a discount for every full month, and the remaining days are billed at day rate.
**_Every month has 22 billable days._**
Help him estimate his cost for such projects, given an hourly rate, the number of days the project spans, and a monthly discount rate.

```javascript

> //TODO: ADD EXAMPLE

Author: Ah, it slipped out of my mind to change the name and remove TODO
